forms-flow-api/src/formsflow_api/models/process.py

```diff
@@ -132,6 +132,18 @@ def filter_conditions(cls, **filters):
         query = cls.query.filter(*filter_conditions) if filter_conditions else cls.query
         return query
 
+    @classmethod
+    def subquery_for_getting_latest_process(cls):
+        """Subquery to get the latest process by parent_process_key."""
+        subquery = (
+            db.session.query(

```

Comment:
Can't we use the order by major version in desc, then by minor version in desc to get the latest version as that's the right way to do it, also it would be easy to do a data fix in case if anything messes up if we follow this

Reply:
okay will do it